OJP/OJP_JourneySupport.xsd

```diff
@@ -599,11 +576,11 @@
 				</xs:element>
 				<xs:element name="TimeAllowance" type="xs:duration" minOccurs="0">
 					<xs:annotation>
-						<xs:documentation>Extra time needed before reaching/after leaving this location (an example of a TRIP ACCESS CONSTRAINT. In Transmodel it is modeled more extensively and could relate to ACCESS MODE e.g. walk max 5 mintues, but cycle 10 minutes).</xs:documentation>
+						<xs:documentation>Extra time needed before reaching/after leaving this location (an example of a TRIP ACCESS CONSTRAINT. In Transmodel it is modeled more extensively and could relate to ACCESS MODE e.g. walk max. 5 minutes, but cycle 10 minutes).</xs:documentation>
 					</xs:annotation>
 				</xs:element>
 			</xs:choice>
-			<xs:element name="IndividualTransportOptions" type="IndividualTransportOptionsStructure" minOccurs="0" maxOccurs="unbounded">
+			<xs:element name="IndividualTransportOptions" type="IndividualTransportOptionStructure" minOccurs="0" maxOccurs="unbounded">
```

Comment:
```suggestion
			<xs:element name="IndividualTransportOption" type="IndividualTransportOptionStructure" minOccurs="0" maxOccurs="unbounded">
```

Reply:
My idea would have been to do it everywhere on the elements as well.. - or are there too many cases dispersed in all the example files? If yes, and we want do do it, a separate PR would probably be better.